### PR TITLE
Update carriers.csv

### PR DESCRIPTION
**Pull request by cmsdroff**
Adding known carrier NMFTA / SCAC codes

### CODE REVIEW

File: datamodel/samples.d/carriers.csv [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-"CMA CGM",CMA,
-"Evergreen Marine Corporation",EMC,
-"Hapag Lloyd",HLC,
-"Hyundai",HMM,
-"Maersk",MSK,
-"Mediterranean Shipping Company",MSC,
-"Ocean Network Express Pte. Ltd.",ONE,
-"Yang Ming Line",YML,
-"Zim Israel Navigation Company",ZIM,
+"CMA CGM",CMA,CMAU
```
Comment (HenrikHL) [inline datamodel/samples.d/carriers.csv:2]: Hi @cmsdroff can you please verify the value for CMA CGM? I have it as `CMDU`

Comment (HenrikHL) [inline datamodel/samples.d/carriers.csv:2]:
```suggestion
"CMA CGM",CMA,CMDU

```

Comment (cmsdroff) [reply datamodel/samples.d/carriers.csv]: it is indeed, my apologies!

**Comment by cmsdroff** [comment]
@HenrikHL I also had notification that Yang Ming will change to `YMJA` for all EDI, the transition period is from the 1st Sept 2023 through to 13st Jan 2024 when both `YMLU` and `YMJA` will be valid, and after that date only `YMJA` will work.

Will leave to you to decide if to update now to `YMJA` or leave as is and change at later date?  

> This is to inform you that the effective date of new SCAC(Standard Carrier Alpha Code) “YMJA” for Yang Ming Joint Service Agreement and associated adjustment on B/L printing format and EDI files will be effective on 2023/10/01.
>What action required from you ?
During transition period: 2023/09/01 to 2024/01/31, YMLU and YMJA are valid in our system.
According to the different settings of EDI Message, there are many different situations that need your clarification and cooperation.

**Comment by HenrikHL** [comment]
Thanks @cmsdroff - we will look into trying to solve that in our internal system ;-)

**Comment by HenrikHL** [comment]
Hi again @cmsdroff - we talked internally about this and will stick to the "old" value and make a PR and switch to the new value beginning of 2024
Thanks for the heads up :-)

**Pull request merged**